packages/breadboard/src/new/grammar/board.ts

```diff
@@ -38,7 +38,7 @@ import { BuilderScope } from "./scope.js";
  * Implementation of the overloaded board function.
  */
 export const board: BoardFactory = (
-  optionsOrFn:
+  optionsOrFn?:
```

Comment:
So, these 2 changes mean that you can now create a completely empty board with `board()` , yeah? Why would we want that?

Reply:
for testing whether a valid board can be created, separate from any potential issues with nodes or wiring.